Assets/SC_Inentory.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Inentory : MonoBehaviour
+{
+    private bool _isFinish;
+
+    private bool salad;
+    private bool _bacon;
+    private bool CHEESE;
+
+    private void Awake()
+    {
+        GameEventSystem.OnPlayerPickUpCollectible += ASD;
+    }
+
+    private void OnDestroy()
+    {
+        GameEventSystem.OnPlayerPickUpCollectible -= ASD;
+    }
+
+    private void Update()
+    {
+        if (!(!salad && !_bacon && !CHEESE))
```

Comment:
This logical expression should have been simplified. I'm having trouble understanding what's going on here